assets/js/main.js

```diff
@@ -1,11 +1,13 @@
 $("body").on({
   mouseenter: function() {
-    if ($('.menu-main .sub-menu').is(':visible')) {
+    var submenu = $(this).find(".sub-menu");
+    if (submenu && submenu[0].children.length) {
       $('.header-main').addClass('header-open');
     }  
   },
   mouseleave: function() {
-    if (!$('.menu-main .sub-menu').is(':visible')) {
+    var submenu = $(this).find(".sub-menu");
+    if (!submenu || submenu[0].children.length === 0 || !submenu.is(':visible')) {
```

Comment:
Y acá es necesario?

Reply:
si porque sino no desaparece

Reply:
mmm, pero en algún momento es se da la condición `!submenu.is(':visible')`? cuando?

Reply:
```
mouseleave: function() {
  $('.header-main').removeClass('header-open');
}
```

Reply:
tenes razon. estoy oxidado de jquery ja